src/lib.rs

```diff
@@ -571,8 +606,29 @@ fn json_obj_len(ctx: &Context, args: Vec<String>) -> RedisResult {
 /// MEMORY <key> [path]
 /// HELP
 ///
-fn json_debug(_ctx: &Context, _args: Vec<String>) -> RedisResult {
-    Err("Command was not implemented".into())
+fn json_debug(ctx: &Context, args: Vec<String>) -> RedisResult {
+    if args.len() < 2 {
```

Comment:
You keep adding these `args.len()` checks... they are unneeded, since the `WrongArity` error will be returned when calling the `next_string` method.

Reply:
done